Dockerfile.worker

```diff
@@ -1,143 +1,110 @@
-# this gdal image comes with support for FileGDB and has python3.4 already installed
-FROM geometalab/gdal-debian:latest
+# this gdal image comes with support for FileGDB and has python3.6 already installed
+# is derived from ubuntu
+FROM geometalab/gdal-docker:v3.0.0
 
 USER root
 
-ENV PYTHONUNBUFFERED=non-empty-string PYTHONIOENCODING=utf-8 LC_ALL=C.UTF-8 LANG=C.UTF-8
+ENV PYTHONUNBUFFERED=non-empty-string PYTHONIOENCODING=utf-8 LC_ALL=C.UTF-8 LANG=C.UTF-8 DEBIAN_FRONTEND=noninteractive
```

Comment:
Should we reset `DEBIAN_FRONTEND` at the end of the Dockerfile? (I think it'll retain the value even into the running containers, if we don't. That might be unexpected when using apt in such a container interactively, e.g. for debugging.)